uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Hook/PageRun/Resources.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+class CRM_Tasksassignments_Hook_PageRun_Resources {
+
+  private $resources;
+
+  /**
+   * @param CRM_Core_Resources $resources
+   */
+  public function __construct(CRM_Core_Resources $resources) {
+    $this->resources = $resources;
+  }
+
+  /**
+   * Adds resources needed by specific Task and Workflow pages.
+   *
+   * @param CRM_Core_Page $page
+   */
+  public function handle($page) {
+    if (!$this->shouldHandle($page)) {
+      return;
+    }
+
+    $this->resources->addScriptFile('uk.co.compucorp.civicrm.tasksassignments',
+        CRM_Core_Config::singleton()->debug
```

Comment:
I think conditionals inside arguments make it less readable, can you maybe make `CRM_Core_Config::singleton()->debug` a variable like `isDebug` or `isDevEnvironment` and also use this to decide which script file to use in a separate line (not inside the `addScriptFile` arguments), e.g.

```php
$scriptFile = $isDebug ?  'js/src/tasks-assignments.js' : 'js/dist/tasks-assignments.min.js';
 $this->resources->addScriptFile(self::EXTENSION_KEY, $scriptFile);
```